### PR TITLE
Add Coins of Interest feature.

### PR DESCRIPTION
**Pull request by reSteiner10**

### CODE REVIEW

File: WebhookBot/index.js [hunk 1]
```diff
@@ -146,9 +146,19 @@ function constructDiscordMessage(configObj, coinmarketcapJSON){
...
+        interestMessage = interestMessage.concat("\n- " + coinmarketcapJSON[i].name + " (" + coinmarketcapJSON[i].percent_change_24h + "%)");
+      };
+    };
+    messageArray.push(interestMessage);
+  };
+
   for (var i = 0, len = messageArray.length; i < len; i++) {
     message = message.concat(messageArray[i] + "\n\n");
-  }
+  };
```
Comment (varzaman) [inline WebhookBot/index.js:172]: extra space

File: WebhookBot/index.js [hunk 2]
```diff
@@ -146,9 +146,19 @@ function constructDiscordMessage(configObj, coinmarketcapJSON){
     messageArray.push("Top Five 24 Hour Gainers: \n1. " + topFive[0].name + " (+%" + topFive[0].percent_change_24h + ")\n2. " + topFive[1].name + " (+%" + topFive[1].percent_change_24h + ")\n3. " + topFive[2].name + " (+%" + topFive[2].percent_change_24h + ")\n4. " + topFive[3].name + " (+%" + topFive[3].percent_change_24h + ")\n5. "+ topFive[4].name + " (+%" + topFive[4].percent_change_24h + ")");
```
Comment (varzaman) [inline WebhookBot/index.js]: Line is way too long. Cut it down boi.

**Review by varzaman — CHANGES_REQUESTED** [review]

File: WebhookBot/index.js (hunk 2 above)
Comment (reSteiner10) [reply WebhookBot/index.js]: Fixed

File: WebhookBot/index.js (hunk 1 above)
Comment (reSteiner10) [reply WebhookBot/index.js:172]: I don't see the extra space.

Comment (varzaman) [reply WebhookBot/index.js:172]: Yea I'm just dumb lol.

**Review by varzaman — APPROVED** [review]

**Pull request merged**